gradle/native_mpp.gradle

```diff
@@ -17,6 +17,7 @@ kotlin {
     }
 
     targets {
+        return
```

Comment:
Accidental change?

Reply:
Since used here kotlin compiler is built from custom branch (because of required changes in the plugin), Kotlin/Native is temporary incompatible